File: simple-client/cpp/simple-client.cpp
#include "open62541pp/open62541pp.h"
#include "open62541pp/Logger.h"

int main() {
    opcua::Client client;
    client.connect("opc.tcp://localhost:4840");

    opcua::Node node = client.getNode(opcua::VariableId::Server_ServerStatus_CurrentTime);
    const auto dt = node.readValueScalar<opcua::DateTime>();

    opcua::log(client, opcua::LogLevel::Info, opcua::LogCategory::Client,
               dt.format("server date is: %Y-%m-%d %H:%M:%S"));
}

File: simple-client/python/simple-client.py
from asyncua import Client

async def main():
    async with Client(url='opc.tcp://localhost:4840') as client:
        # Do something with client
        node = client.get_node('i=2258')
        value = await node.read_value()
        print(f'Server Time: {value}')


if __name__ == "__main__":
    import asyncio
    asyncio.run(main())
